kDrive/UI/Controller/Files/Preview/PreviewViewController.swift

```diff
@@ -213,13 +213,15 @@ class PreviewViewController: UIViewController, PreviewContentCellDelegate, Scene
 
     func observeFileUpdated() {
         driveFileManager?.observeFileUpdated(self, fileId: nil) { [weak self] file in
-            if self?.currentFile.id == file.id {
-                self?.currentFile = file
-                Task { @MainActor in
-                    guard let self else { return }
-                    self.collectionView.endEditing(true)
-                    self.collectionView.reloadItems(at: [self.currentIndex])
-                }
+            guard let self = self,
```

Comment:
This could be simplified to `guard let self` ?

Reply:
Yup, I wonder why the linter does not autocorrect this.

Reply:
Yup, I will check if it's a rule we need to enable